app/app/controllers/cbv/base_controller.rb

```
@@ -86,31 +86,29 @@ def prevent_back_after_complete
   end
 
   def capture_page_view
-    client = DeviceDetector.new(request.headers["User-Agent"])
-    NewRelicEventTracker.track("CbvPageView", {
-      user_agent: request.headers["User-Agent"],
-      device_name: client.device_name,
-      device_type: client.device_type,
-      browser: client.name,
-      cbv_flow_id: @cbv_flow.id,
-      invitation_id: @cbv_flow.cbv_flow_invitation_id,
-      site_id: @cbv_flow.site_id,
-      path: request.path
-    })
-  rescue => ex
-    Rails.logger.error "Unable to track NewRelic event (CbvPageView): #{ex}"
+    begin
+      event_logger.track("CbvPageView", request, {
+        cbv_flow_id: @cbv_flow.id,
+        invitation_id: @cbv_flow.cbv_flow_invitation_id,
+        site_id: @cbv_flow.site_id,
+        path: request.path
```

Note, I believe this is breaking some views, specifically expired links.

The error message is being swallowed and the user is redirected to a flash error message, which is why we didn't catch it.